packages/client/src/views/SysAdmin/Config/Application/Application.tsx

```diff
@@ -487,168 +168,70 @@ function DeathTabContent({
-    }
-    return <FormTabs {...tabs} />
-  }
-
-  render() {
-    const { intl, offlineCountryConfiguration } = this.props
-
-    return (
-      <SysAdminContentWrapper
-        isCertificatesConfigPage={true}
-        hideBackground={true}
       >
-        <Content
-          title={intl.formatMessage(messages.applicationSettings)}
-          titleColor={'copy'}
-          tabBarContent={this.getTabs(intl)}
-        >
-          {this.state.activeTabId && this.state.activeTabId === TabId.GENERAL && (
-            <GeneralTabContent
-              offlineCountryConfiguration={offlineCountryConfiguration}
-              intl={intl}
-              callBack={(modalName: string) =>
-                this.setState({
-                  changeModalName: modalName
-                })
-              }
-            />
-          )}
-          {this.state.activeTabId && this.state.activeTabId === TabId.BIRTH && (
-            <BirthTabContent
-              offlineCountryConfiguration={offlineCountryConfiguration}
-              intl={intl}
-              callBack={(modalName: string) =>
-                this.setState({
-                  changeModalName: modalName
-                })
-              }
-            />
-          )}
-          {this.state.activeTabId && this.state.activeTabId === TabId.DEATH && (
-            <DeathTabContent
-              offlineCountryConfiguration={offlineCountryConfiguration}
-              intl={intl}
-              callBack={(modalName: string) =>
-                this.setState({
-                  changeModalName: modalName
-                })
-              }
-            />
-          )}
-        </Content>
-        {this.state.changeModalName && (
-          <DynamicModal
-            toggleConfigModal={this.toggleConfigModal}
-            changeModalName={this.state.changeModalName}
-            showNotification={this.state.showNotification}
-            valueChanged={this.changeValue}
-          />
-        )}
-        <FloatingNotification
-          id="print-cert-notification"
-          type={this.state.notificationStatus}
-          show={this.state.showNotification}
-          callback={() => {
-            this.setState({ showNotification: false })
-          }}
-        >
-          {this.state.notificationMessages}
-        </FloatingNotification>
-      </SysAdminContentWrapper>
-    )
-  }
-}
-
-function mapStateToProps(state: IStoreState) {
-  return {
-    userDetails: getUserDetails(state),
-    offlineCountryConfiguration: getOfflineData(state)
-  }
+        {activeTabId === TabId.GENERAL && <GeneralTabContent />}
+        {activeTabId === TabId.BIRTH && <BirthTabContent />}
+        {activeTabId === TabId.DEATH && <DeathTabContent />}
+      </Content>
+    </SysAdminContentWrapper>
+  )
 }
 
-export const ApplicationConfig = connect(mapStateToProps)(
-  injectIntl(ApplicationConfigComponent)
-)
+export const ApplicationConfig = ApplicationConfigComponent
```

Comment:
We can just rename `ApplicationConfigComponent` to `ApplicationConfig`

Reply:
Done